rubric_visualization/settings/aws.py

```diff
@@ -14,18 +14,6 @@
 REDIS_HOST = SECURE_SETTINGS.get('redis_host', '127.0.0.1')
 REDIS_PORT = SECURE_SETTINGS.get('redis_port', 6379)
 
-CACHES = {
```

Comment:
Just curious why the redis cache settings were removed.

Reply:
If you see `from .base import *` at the top of `aws.py`, that means that `aws.py` inherits all of `base.py`. The caches settings were declared in `base.py`, so they are safe to remove from `aws.py` 🙂

Reply:
ohh gotcha!